test/test_parallel.jl

```diff
@@ -208,17 +220,17 @@ import ParallelStencil.@gorgeousexpand
                         return
                     end
                     @parallel write_indices!(A);
-                    @test all(Array(A) .== [ix + (iy-1)*size(A,1) + (iz-1)*size(A,1)*size(A,2) for ix=1:size(A,1), iy=1:size(A,2), iz=1:size(A,3)])
+                    @test all(Array(A) .≈ [ix + (iy-1)*size(A,1) + (iz-1)*size(A,1)*size(A,2) for ix=1:size(A,1), iy=1:size(A,2), iz=1:size(A,3)])
                 end
                 @testset "@parallel <kernel> (3D; on-the-fly)" begin
                     nx, ny, nz = 32, 8, 8
-                    lam=dt=_dx=_dy=_dz = 1.0
+                    lam=dt=_dx=_dy=_dz = 1
                     T      = @zeros(nx, ny, nz);
                     T2     = @zeros(nx, ny, nz);
                     T2_ref = @zeros(nx, ny, nz);
                     Ci     = @ones(nx, ny, nz);
                     copy!(T, [ix + (iy-1)*size(T,1) + (iz-1)*size(T,1)*size(T,2) for ix=1:size(T,1), iy=1:size(T,2), iz=1:size(T,3)].^3);
-                    @parallel function diffusion3D_step!(T2, T, Ci, lam::Data.Number, dt::Float64, _dx, _dy, _dz)
+                    @parallel function diffusion3D_step!(T2, T, Ci, lam, dt, _dx, _dy, _dz)
```

Comment:
I don't remember why `lam::Data.Number, dt::Float64` was there but there must have been some reason. It could also be related to them bitwise identical results

Reply:
Or maybe it was just a test that this type specification in the signature works

Reply:
Reverted the change in f7d1d7471b39a914a19266456e2923c6fda627c9